frontend/src/pages/Home.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState, useEffect } from 'react';
+import ProductCard from '../components/ProductCard';
+import Header from '../components/Header';
+import SideBar from '../components/SideBar';
+import Pagination from '../components/Pagination';
+import { useLazyGetProductsQuery } from '../services/products';
+
+const Home = () => {
```

Comment:
I like the distinction between components and pages, and that you have the API in a service. Not sure if you looked at some example projects or just have amazing instincts, but this structure is much more professional than what's typically submitted at your level. Whatever you're doing, keep doing it!